docs/esp32/quickref.rst

```diff
@@ -189,6 +189,33 @@ Use the ``machine.PWM`` class::
 
     pwm2 = PWM(Pin(2), freq=20000, duty=512) # create and configure in one go
 
+MCPWM (motor control pulse width modulation)
+--------------------------------------------
+
+MCPWM can be enabled on all output-enabled pins. The API differs slightly from `machine.PWM` to allow explicit timer selection.
+Currently, this implementation only covers basic functionality.  
+
+For more details, see Espressif's `MCPWM
+<https://docs.espressif.com/projects/esp-idf/en/latest/api-reference/peripherals/mcpwm.html>`_ documentation.
+
+
+Use the `esp32.MCPWM` class::
+
+    from machine import Pin
+    from esp32 import MCPWM
+
+    pwm0 = PWM(0)           # Create MCPWM object with timer ID (0..5)
```

Comment:
Should be `MCPWM` not `PWM`.

Also the docs (and MCPWM_TIMER_MAX in mcpwm.h) says that it can be 0,1,2

Reply:
Agreed, should be MCPWM! 
However, the MCPWM object abstracts _both_ ESP32 MCPWM units, where timer ids 0..2 are on unit 1, and ids 3..5 on unit 2